### PR TITLE
Fix zenhub burndown page

### PR DESCRIPTION
**Pull request by stedesign**
I've tried to fix the Burndown page for Zenhub, below the result

fix

### CODE REVIEW

**Comment by Mottie** [comment]
Hi @stedesign!

Thanks for your contribution!

Notes on this PR:

* As much as possible, please combine definitions with close-to or similar colors.
* All newly added definitions should be appended to the list of selectors (next to the open curly bracket) so we can get a general idea of which selectors may have changed and may be removed.
* Lastly, there is no need to add yourself to the authors file as the file is automatically generated 😸

File: github-dark.css [hunk 1]
```diff
@@ -2371,8 +2371,8 @@
   .render-shell img:not(.math) {
     background-image: url("data:image/gif;base64,R0lGODlhCgAKAIAAABoaGgAAACwAAAAACgAKAAACEYQdmYcaDNxjEspKndVZbc8UADs=") !important;
   }
-  .render-shell:hover .img-view img, .file:hover .image img, .file:hover .image canvas, img.asset:hover,
-  .render-shell:hover img:not(.math) {
+  .render-shell:hover .img-view img, .file:hover .image img,
+  .file:hover .image canvas, img.asset:hover, .render-shell:hover img:not(.math) {
```
Comment (Mottie) [inline github-dark.css]: Please add all new selectors next to the `{`, instead of before other (older) selectors.

File: github-dark.css [hunk 2]
```diff
@@ -2733,6 +2737,19 @@
...
+  .zhc-progress-bar > .zhc-progress-bar--completed-issues {
+    background: #163 !important;
+  }
+  .zhc-item-table__header {
+    color: #e2e2e2 !important;
+    background: #202020 !important;
+  }
+  .zhc-item-table__cell__issue-number {
+    color: #7b7b7b !important;
+  }
```
Comment (Mottie) [inline github-dark.css]: While we are keeping the ZenHub definitions separate from the main page definitions, please try to combine these new additions with other ZenHub definitions... with same, or nearly-similar colors. Thanks!

File: github-dark.css [hunk 3]
```diff
@@ -2743,7 +2760,8 @@
   .zh-epic-creator-nav, .zh-epic-creator-picker-container, .zh-board-menu,
   .zh-epic-creator-filters, .zh-epic-issues-picker-new-issue, .merge-status-list,
   .zhc-dropdown-simple-item, .zhc-pipeline-body, .zhc-board-settings-head,
-  .zhc-pipeline-sort-status {
+  .zhc-pipeline-sort-status, .zhc-chart-container, .zhc-burndown-chart__stats,
+  .zhc-chart-container__footer {
     border-color: #343434 !important;
```
Comment (Mottie) [inline github-dark.css]: Perfect! 😉

File: github-dark.css (hunk 1 above)
Comment (stedesign) [inline github-dark.css]: I haven't touched those classes... I think `grunt clean` made this change

Comment (Mottie) [inline github-dark.css]: Oh, you're right... it shifted the selectors down. Sorry.

**Comment by Mottie** [comment]
Thanks again!

**Pull request merged**